pykrige/ok.py

```diff
@@ -439,12 +439,20 @@ def print_statistics(self):
         print("Q2 =", self.Q2)
         print("cR =", self.cR)
 
-    def _get_kriging_matrix(self, n):
+    def _get_kriging_matrix(self, n, xy=None, coordinates='euclidean'):

```

Comment:
We can use self.coordinates_type instead of passing coordinates

Reply:
This sounds correct.